raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -58,7 +58,12 @@ library ChannelManagerLibrary {
 
         address[] storage existingChannels = self.nodeChannels[msg.sender];
```

Comment:
I don't know for sure if the code is correct or not. At the moment I'm not able to run solidity code (like in I don't know how to do it).

Can you confirm that this isn't an issue?